src/entry/data-provider/index.ts

```diff
@@ -31,6 +35,32 @@ export const dataProvider = async (request: DataProviderPayload): Promise<DataPr
 
   const trackingBranch = await getTrackingBranchName(groupToken, projectId, defaultBranch);
 
+  const backfillData: BackfillData = {
```

Comment:
Do you need to initialize this with values? Could we just do `const backfillData: BackfillData;`?

Reply:
We have typescript strict mode enabled so we will get an error. We can make the variable `let`, but we are not reassigning the whole variable, just the properties.